src/interaction.js

```diff
@@ -41,6 +41,7 @@ export const DEFAULT_AUTO_INSTRUMENTED_EVENTS = {
 export class SplunkUserInteractionPlugin extends UserInteractionPlugin {
   constructor({adjustAutoInstrumentedEvents}) {
     super();
+    this._tracerName = 'route';
```

Comment:
What is the story behind this change?

Reply:
I can't reach `window.SplunkRum.provider`, because the code might be running in CJS. `import SplunkRum from './index'` will cause a circular dependency. So I changed to rely on BasePlugin.